ceasiompy/SU2Run/func/su2actuatordiskfile.py

```diff
@@ -59,11 +61,9 @@ def get_radial_stations(radius, hub_radius, number_of_stations=40):
     if hub_radius >= radius:
         raise ValueError("hub radius should be smaller than radius")
 
+    hub_radius_ratio = hub_radius / radius
```

Comment:
Maybe not useful to define that `hub_radius / radius` could be use directly in the next line